Source/Core/Core/IOS/SDIO/SDIOSlot0.cpp

```diff
@@ -289,6 +324,9 @@ s32 SDIOSlot0::ExecuteCommand(const Request& request, u32 _BufferIn, u32 _Buffer
     Memory::Write_U32(0x900, _BufferOut);
     break;
 
+  // The following events don't seem to be possible due to the command length being only 6 bits
+  // I don't want to delete this without more context on why it's there and what it's meant to do
+  /*
   case EVENT_REGISTER:  // async
```

Comment:
I was hoping someone could clarify. I could not find any docs, and I don't think it actually does nything in the case of Dolphin.
When I was working on this (months ago) I noticed that games did not attempt to initialize SD at all.

Since you mentioned this, my suspicion is that this is supposed to be a combined CMD0 and event register, rather than just the event register.
I'll probably do some debugging in the week-end.